packages/manager/apps/telecom/src/app/telecom/telephony/alias/administration/terminate/terminate.controller.js

```diff
@@ -1,105 +1,140 @@
-export default /* @ngInject */ function TelecomTelephonyAliasAdministrationTerminateCtrl(
-  $q,
-  $stateParams,
-  $translate,
-  OvhApiTelephony,
-  TelephonyMediator,
-  TucToast,
-  TucToastError,
-) {
-  const self = this;
+import confirmTemplate from './confirm/confirm.html';
+import controller from './confirm/confirm.controller';
 
-  function getTerminationReasons() {
-    return TelephonyMediator.getApiScheme().then(
+export default /* @ngInject */ class TerminateController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    $uibModal,
+    OvhApiTelephony,
+    TelephonyMediator,
+    TucToast,
+    TucToastError,
+    tucVoipServiceAlias,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.TelephonyMediator = TelephonyMediator;
+    this.TucToast = TucToast;
+    this.TucToastError = TucToastError;
+    this.tucVoipServiceAlias = tucVoipServiceAlias;
+    this.$uibModal = $uibModal;
+  }
+
+  $onInit() {
+    this.isLoading = true;
+    this.$q
+      .all({
+        reason: this.getTerminationReasons(),
```

Comment:
consistency mistake, we call a function which return reasons, may be we can name attribute `raisons` ?

```suggestion
        reasons: this.getTerminationReasons(),
```